moviepy/video/VideoClip.py

```diff
@@ -1544,11 +1544,11 @@ class TextClip(ImageClip):
       letters with accents), and bellow standard baseline (e.g letters such as
       p, y, g).
 
-      This notion is knowned under the name ascent and descent meaning the
-      highest and lowest pixel above and below baseline
+      This notion is known under the name ascent and descent meaning the
+      highest and lowest pixel above and below the baseline.
 
-      If your first line dont have an "accent character" and your last line
-      dont have a "descent character", you'll have some "fat" arround
+      If your first line doesn't have an "accent character" and your last line
+      doesn't have a "descent character", you'll have some "fat" arround.
```

Comment:
I assume the last word should read "around" – would be neat if you could correct that as well.